lib/printers/pretty.js

```diff
@@ -55,6 +55,24 @@ class PrettyPrinter {
         this.console.log(chalk.red(chalk.bold(fixableSummary)));
       }
     }
+
+    if (this.options.updateTodo && todoInfo) {
+      let todoSummary = `✔ ${todoInfo.added} todos created `;
+
+      if (todoInfo.todoConfig) {
+        let todoConfig = todoInfo.todoConfig;
+
+        if (todoConfig.warn && todoConfig.error) {
+          todoSummary += `(warn after ${todoConfig.warn} and error after ${todoConfig.error} days)`;
+        } else if (todoConfig.warn) {
+          todoSummary += `(warn after ${todoConfig.warn} days)`;
+        } else if (todoConfig.error) {
+          todoSummary += `(error after ${todoConfig.error} days)`;
+        }
```

Comment:
I think I'd prefer to avoid the "and" between the two (when both error and warn are present), replacing it either with a comma or semicolon.

I think that may also allow us to simplify the logic slightly, but not sure (building end user helpful prose is always slightly annoying 😝 ).

What do you think?

Reply:
Yep. This building was done simply to accommodate the `and`, so removing it will for sure simplify.

Reply:
Done.